examples/client-benchmark/main.cpp

```diff
@@ -388,14 +385,14 @@ shared_ptr<PeerConnection> createPeerConnection(const Configuration &config,
 						break;
 
 					try {
-						float dataCountMultiplier =
-						    duration_cast<microseconds>((steady_clock::now() - stepTime)).count() /
-						    (1000.0 * stepDurationInMs);
+						const double elapsedTimeInMs =
+						    std::chrono::duration<double>(steady_clock::now() - stepTime).count() *
```

Comment:
Here it should be `std::chrono::duration<milliseconds>`, otherwise you are not sure about the unit.

Reply:
Here I need duration as double. `std::chrono::duration<double>` is in seconds so I am converting to ms by multipling with 1000. Am I missing something?

Reply:
Oh OK, my bad. Could you just change the variable name to `elapsedTimeInSecs` to reflect it's now in seconds? After that it looks good.

Reply:
Actually it is ms. I moved multiplication with 1000 to lower line for you :)
So now it is `elapsedTimeInSecs`.